console/src/components/UIKit/atoms/Tooltip/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import Tooltip from 'react-bootstrap/lib/Tooltip';
+import { StyledTooltip } from './Tooltip';
+
+export type ToolTypeGeneratorProps = {
+  message: string;
+};
+
+export const tooltipGenerator: React.FC<ToolTypeGeneratorProps> = ({
+  message,
+}) => {
+  return <Tooltip id={message}>{message}</Tooltip>;
+};
+
+export type ToolTipProps = {
+  message: ToolTypeGeneratorProps;
```

Comment:
We want to pass `message` as a string.
```suggestion
  message: string;
```